genericclient_aiohttp/__init__.py

```diff
@@ -208,7 +208,9 @@ def get_or_create_session(self):
         return self
 
     async def __aexit__(self, *args, **kwargs):
-        await self._session.__aexit__(*args, **kwargs)
+        self.get_or_create_session()
```

Comment:
this line need to be moved to the `__aenter__` method

Reply:
Doh yeah typo